Update AlphaAnimals_CE_Patch_Race_Agaripod.xml

Breadbox2k19 opened the pull request.

Breadbox2k19 committed:
Update AlphaAnimals_CE_Patch_Race_Agaripod.xml

Juanfrank committed:
Update AlphaAnimals_CE_Patch_Race_Agaripod.xml

Juanfrank commented:
I think armor values should be switched around since the Agaripod is "almost impervious to gunfire, with bullets simply being absorbed into its mass without doing much damage, but very weak to fire". This was my mistake.

If Bullets are sharp damage, then it{s all the way around. This also applies to Wildpawns-pods.

Breadbox2k19 commented:
I don't have much of an opinion on balancing of the various pawns and pods since they aren't exactly combat animal imo. Feel free to make any adjustments to them.

Juanfrank committed:
Update AlphaAnimals_CE_Patch_Race_Agaripod.xml

Sharp armor just because that's ammo's main characteristic. Pods are "meant" to make guns fodder-ish, in favour of using fire.
